packages/documentation/migrations/2_ETH_FAUCET.js

```diff
@@ -0,0 +1,11 @@
+/* global artifacts */
+const EthFaucet = artifacts.require('./EthFaucet.sol');
+
+module.exports = (deployer, network) => {
+    deployer.deploy(
+        EthFaucet,
+        '0xA1e862D85419a57D588CD4566e558f5987cFa67E',
+        // TODO change to be from env
```

Comment:
Is this TODO something we still want to do?

Reply:
We should really move the faucet to be part of the protocol, I just deployed it in Jan when we launched. I am not sure if its a huge priority though as its not really going to change

Reply:
Why move it to protocol? It's not a part of the protocol, and think we risk making that package a bit crowded (especially since we've already moved things like account registry in it). If the main reason is to be able to test things more easily, I'd argue we should break out our migrations/testing out of protocol instead?

Not a huge priority though.